utils/vectorscan/Makefile

```diff
@@ -0,0 +1,92 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vectorscan
+PKG_VERSION:=5.4.9
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/VectorCamp/vectorscan/tar.gz/$(PKG_NAME)/$(PKG_VERSION)?
+PKG_HASH:=e61c78f26a9d04ccffab0df1159885c4503fc501172402c57f7357a2126ea3c6
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=John Audia <therealgraysky@proton.me>
+PKG_LICENSE:=BSD-3-Clause BSD-2-Clause BSL-1.0
+PKG_LICENSE_FILES:=LICENSE
+PKG_BUILD_PARALLEL:=1
+CMAKE_INSTALL:=1
+
+PKG_BUILD_DEPENDS:=ragel/host python3/host boost/host
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+CMAKE_OPTIONS += \
+	-DCMAKE_INSTALL_LIBDIR=lib \
+	-DBUILD_SHARED_LIBS=ON \
+	-Wno-dev
+
+define Package/vectorscan-full
+  CATEGORY:=Utilities
+  SECTION:=utils
+  TITLE:=Vectorscan Full
+  URL:=https://github.com/VectorCamp/vectorscan
+  DEPENDS:=@(aarch64||arm||i386||i686||x86_64)
+endef
+
+define Package/vectorscan-runtime
+  CATEGORY:=Utilities
+  SECTION:=utils
+  TITLE:=Vectorscan Runtime
+  URL:=https://github.com/VectorCamp/vectorscan
+  DEPENDS:=@(aarch64||arm||i386||i686||x86_64)
+endef
+
+define Package/vectorscan-full/description
+  A fork of Intel's Hyperscan, modified to run on more platforms. Currently ARM
+  NEON/ASIMD is 100% functional, and Power VSX are in development. ARM SVE2
+  will be implemented when harwdare becomes accessible to the developers.
+  More platforms will follow in the future, on demand/request.
+  
+  Contains shared objects and headers.
+endef
+
+define Package/vectorscan-runtime/description
+  A fork of Intel's Hyperscan, modified to run on more platforms. Currently ARM
+  NEON/ASIMD is 100% functional, and Power VSX are in development. ARM SVE2
+  will be implemented when harwdare becomes accessible to the developers.
+  More platforms will follow in the future, on demand/request.
+  
+  Contains shared objects only.
+endef
+
+define Package/vectorscan-runtime/install
+endef
```

Comment:
I'm going to stop reviewing this PR now because I'm not interested in writing this package for you.